api/app/src/mappings/dexcom/biometrics.ts

```diff
@@ -0,0 +1,52 @@
+import { Biometrics } from "@metriport/api";
+import { Sample } from "@metriport/api/lib/devices/models/common/sample";
+
+import { DexcomEvgs } from "./models/evgs";
+import { PROVIDER_DEXCOM } from "../../shared/constants";
+import { Util } from "../../shared/util";
+
+export const mapToBiometrics = (dexcomBiometrics: DexcomEvgs, date: string): Biometrics => {
+  const metadata = {
+    date: date,
+    source: PROVIDER_DEXCOM,
+  };
+
+  const biometrics: Biometrics = {
+    metadata: metadata,
+  };
+
+  const samples = dexcomBiometrics.records.reduce((acc: Sample[], curr) => {
```

Comment:
NABD, but the type of accumulator its inferred from the initialization, which we can use `new Array<Sample>()`. Just mentioning this bc I think its better to have types based on initialization than the other way around - you can see if you remove `: Sample[]` from the `acc` param and hover it, its `never`, which means we're setting the type of a param that receives a value/obj without type, so the compiler can't help us if they mismatch.